src/snowflake/connector/errors.py

```diff
@@ -161,19 +165,27 @@ def send_exception_telemetry(
             telemetry_data[TelemetryField.KEY_EXCEPTION.value] = self.__class__.__name__
```

Comment:
@sfc-gh-fpawlowski @sfc-gh-pczajka should we add additional information about aio usage? At this point we can check class of connection to determine if aio is being used or not. WDYT?